frontend/cypress/integration/Error.cy.ts

```diff
@@ -27,6 +27,6 @@ describe('Error boundary behaviors', () => {
         cy.get('[data-cy=error-button]').click();
 
         // Bullet should be gone
-        cy.get('[data-cy=main-pannel]').children().should('have.length', 0);
```

Comment:
cy.get('[data-cy=main-pannel]').children() didn't work because cy.get('[data-cy=main-pannel]') had a child, the scroll bar. 

Now we directly count the bullets / scripts rows